third_party/openvino/ngraph_c_api/src/ngraph_c_api.cpp

```diff
@@ -50,7 +52,7 @@
   CREATE_NODE_AND_CATCH_EXCEPTIONS(impl, node)
 
 struct ngraph_node {
```

Comment:
Can we name the structure 'ngraph_output' or 'ngraph_output_node'?

Reply:
That's a good discussion. I don't want to export `ngraph_output_node_t `c struct that will confuse to read code, so we consider the `ngraph_node_t `is output node, but it's still an open issue.
```
IEStatusCode ngraph_variadic_split(const ngraph_node_t* input,
                                   const ngraph_node_t* axis,
                                   const ngraph_node_t* splits,
                                   ngraph_node_t** node) {
```

=>

```
IEStatusCode ngraph_variadic_split(const ngraph_output_node_t * input,
                                   const ngraph_output_node_t * axis,
                                   const ngraph_output_node_t * splits,
                                   ngraph_output_node_t ** node) {
```

Reply:
Indeed, it is easy to confuse code reading. Maybe we can find a suitable way to sort out the naming in the following discussing.

Reply:
Great, thanks. merge it first.